metriq_gym/dispatch_qv.py

```diff
@@ -21,6 +20,8 @@ def main():
         f"Dispatching Quantum Volume job with n={args.num_qubits}, shots={args.shots}, trials={args.trials}, backend={args.backend}, confidence_level={args.confidence_level}, jobs_file={args.jobs_file}"
     )
 
+    job_manager = JobManager(args.jobs_file)
```

Comment:
Do we want to make use of the job manager construct in the CLOPS dispatch as well?